haskell-src/Concordium/ID/Types.hs

```diff
@@ -295,6 +295,12 @@ instance Ord CredentialRegistrationID where
 instance FromJSON CredentialRegistrationID where
   parseJSON = withText "Credential registration ID in base16" deserializeBase16
 
+newtype RawCredentialRegistrationID = RawCredRegId (FBS.FixedByteString RegIdSize)
+   deriving newtype (Eq, Ord, Show)
```

Comment:
It would be more useful to display (Show) using hex. For this purpose we have the `FBSHex` auxiliary type. So
```
deriving Show via (FBSHex RegIdSize)
```